openid_connect_configuration.conf

```diff
@@ -63,11 +67,13 @@ proxy_cache_path /var/cache/nginx/jwk levels=1 keys_zone=jwk:64k max_size=1m;
 # Change timeout values to at least the validity period of each token type
 keyval_zone zone=oidc_id_tokens:1M state=conf.d/oidc_id_tokens.json timeout=1h;
 keyval_zone zone=refresh_tokens:1M state=conf.d/refresh_tokens.json timeout=8h;
+keyval_zone zone=pkce:1M state=conf.d/pkce.json timeout=2m;
```

Comment:
Do we need a state file for pkce? Suggest separating this from the other keyvals because as ask the customer to adjust the timeouts.

Reply:
agreed! Will change this.